pulsar-broker/src/test/java/org/apache/pulsar/client/impl/NegativeAcksTest.java

```diff
@@ -255,10 +255,17 @@ public void testNegativeAcksWithBackoff(boolean batching, boolean usePartitions,
         final int redeliverCount = 5;
         long firstReceivedAt = System.currentTimeMillis();
         long expectedTotalRedeliveryDelay = 0;
+        Message<String> msg = null;
```

Comment:
I think that this change is not relevant. In a good coding style it's better to reduce the scope of variables to minimal instead of reusing variables for different purposes.

Reply:
Updated.